ts/features/fci/components/ErrorComponent.tsx

```diff
@@ -1,41 +1,55 @@
 import * as React from "react";
 import { SafeAreaView } from "react-native";
+import { EmailString } from "@pagopa/ts-commons/lib/strings";
 import I18n from "../../../i18n";
 import { IOStyles } from "../../../components/core/variables/IOStyles";
-import { InfoScreenComponent } from "../../../components/infoScreen/InfoScreenComponent";
 import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
 import FooterWithButtons from "../../../components/ui/FooterWithButtons";
-import { cancelButtonProps } from "../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
 import { renderInfoRasterImage } from "../../../components/infoScreen/imageRendering";
 import { WithTestID } from "../../../types/WithTestID";
+import { InfoScreenComponent } from "./InfoScreenComponent";
 
 type Props = WithTestID<{
   title: string;
   subTitle: string;
   image: number;
+  email?: EmailString;
+  retry?: boolean;
   onPress: () => void;
 }>;
 
-const ErrorComponent = (props: Props) => (
-  <BaseScreenComponent goBack={false}>
-    <SafeAreaView style={IOStyles.flex} testID={props.testID}>
-      <InfoScreenComponent
-        image={renderInfoRasterImage(props.image)}
-        title={props.title}
-        body={props.subTitle}
-      />
+const ErrorComponent = (props: Props) => {
+  const closeButtonProps = {
+    block: true,
+    primary: true,
+    bordered: true,
+    onPress: props.onPress,
+    title: I18n.t("global.buttons.close")
+  };
+  const retryButtonProps = {
+    block: true,
+    primary: true,
+    onPress: props.onPress,
+    title: I18n.t("global.buttons.retry")
+  };
```

Comment:
What do you think writing something like that?

 ```ts
const buttonProps = {
    block: true,
    primary: true,
    onPress: props.onPress
  };

  const closeButtonProps = {
    ...buttonProps,
    bordered: true,
    title: I18n.t("global.buttons.close")
  };

  const retryButtonProps = {
    ...buttonProps,
    title: I18n.t("global.buttons.retry")
  };
```
We could improve code readability by separating between "standard" props and the one specific for "close" or "retry" buttons.